sdk/servicebus/azure-servicebus/azure/servicebus/_common/utils.py

```diff
@@ -222,20 +214,39 @@ def transform_messages_to_sendable_if_needed(messages):
         except AttributeError:
             return messages
 
+
+def _single_message_from_dict(message, message_type):
+    # type: (Union[ServiceBusMessage, Mapping[str, Any]], Type[ServiceBusMessage]) -> ServiceBusMessage
+    if isinstance(message, message_type):
+        return message
+    try:
+        return message_type(**cast(Mapping[str, Any], message))
+    except TypeError as e:
+        if "required keyword arguments" in str(e):
+            raise e
```

Comment:
oops, I think this is added in the wrong place :p

should be in the `sdk/servicebus/azure-servicebus/azure/servicebus/management/_utils.py`

Reply:
Hahahahaha thank you thank you :D